api/v2.0/swagger.yaml

```diff
@@ -2044,23 +2055,23 @@ parameters:
     type: string
     required: false
     minLength: 1
+  isResourceName:   
+    name: X-Is-Resource-Name
+    description: The flag to indicate whether the parameter in the path is the name of the resource
+    in: header
+    type: boolean
+    required: false
+    default: false
   projectName:
     name: project_name
     in: path
     description: The name of the project
     required: true
     type: string
-  projectId:
-    name: project_id
-    in: path
-    description: The ID of the project
-    required: true
-    type: integer
-    format: int64
-  projectIDOrName:
-    name: project_id_or_name
+  projectNameOrId:
+    name: project_name_or_id
     in: path
-    description: The id or name of the project
+    description: The name or id of the project
```

Comment:
Why id or name -> name or id?

Reply:
Because the parameter renamed from `projectIdOrName` to `projectNameOrId`, and we prefer the callers to use the name of the project in the APIs.